src/Movies/Movies.js

```diff
@@ -1,17 +1,37 @@
 import React from 'react';
 import MovieCard from '../Movie-Card/MovieCard';
+import './Movies.css'
+import ShowDetails from '../Details/ShowDetails'
+
+const Movies = ({ movieData, movieSelected, handleClick }) => {
+
+  const movieDisplay = () => {
+    if(movieSelected) {
+      const selection = movieData.find(movie => movie.id === movieSelected)
```

Comment:
I made a note that we need to keep the iterator methods outside of our component in cases like these